landing-page/components/sections/ContactSection.tsx

```diff
@@ -8,8 +8,49 @@ import { Toast } from '../Toast'
 import Send from '../../icons/Send'
 import AlertCircle from '../../icons/AlertCircle'
 import { hubspotSubmitForm } from '../../utils/hubspot'
+import { useTranslation } from '../../hooks/useTranslation'
+import type { PropsWithLanguage } from '../../hooks/useTranslation'
+import type { Languages } from '../../hooks/useLanguage'
 
-const ContactSection = forwardRef<HTMLDivElement>(function ContactSection(_, ref) {
+type ContactSectionLanguage = {
+  heading: string,
+  yourInformation: string,
+  email: string,
+  firstName: string,
+  lastName: string,
+  message: string,
+  send: string,
+  successMessage: string,
+  failureMessage: string,
+}
+
+const defaultContactSectionTranslations: Record<Languages, ContactSectionLanguage> = {
+  en: {
+    heading: 'Contact',
+    yourInformation: 'Your Information',
+    email: 'Email',
+    firstName: 'First name',
+    lastName: 'Last name',
+    message: 'Message',
+    send: 'Send',
+    successMessage: 'Message sent!',
+    failureMessage: 'Something went wrong!'
+  },
+  de: {
+    heading: 'Kontakt',
+    yourInformation: 'DE: Your Information', // TODO: translate (in a nice sounding way; not "Ihre/Deine Informationen")
+    email: 'Email',
+    firstName: 'Vorname',
+    lastName: 'Nachname',
+    message: 'Nachricht',
+    send: 'Abschicken',
+    successMessage: 'Nachricht gesendet!',
+    failureMessage: 'Es ist ein Fehler aufgetreten!'
+  }
+}
```

Comment:
Wouldn`t it make sense that we collect these translations in `.json` file for each language, so that someone can add a translations without long searching?

Reply:
This effects more than just this few loc.

Reply:
I kinda want the translations to be scoped / bound to the concrete component.

Additionally there isn't only json data in the translations but also full on react components and functions and dealing with json is quite annoying when it comes to bundling and type checking